multiqc/modules/qualimap/QM_BamQC.py

```diff
@@ -547,6 +548,15 @@ def general_stats_headers (self):
         'shared_key': 'read_count',
         'hidden': True
     }
+    self.general_stats_headers['general_error_rate'] = {
+        'title': 'Error rate'.format(config.read_count_prefix),
```

Comment:
You can remove the `.format(config.read_count_prefix)` bit - that's for adding in the 'M' for million etc. eg:

```python
'title': 'Error rate',
```